riscv-asm.md

```diff
@@ -693,8 +693,8 @@ Load and Store Global
 The following pseudoinstructions are available to load from and store to
 global objects:
 
-  * `l{b|h|w|d} <rd>, <symbol>`: load byte, half word, word or double word from global[^1]
-  * `s{b|h|w|d} <rd>, <symbol>, <rt>`: store byte, half word, word or double word to global[^2]
+  * `l{b|bu|h|hu|w|wu|d} <rd>, <symbol>`: load byte, half word, word or double word from global[^1]
+  * `s{b|bu|h|hu|w|wu|d} <rd>, <symbol>, <rt>`: store byte, half word, word or double word to global[^2]
```

Comment:
There are no unsigned stores (just loads).

Reply:
Yes, sorry for this.